src/backend/serial/scalar_mul/abglsv_pornin.rs

```diff
@@ -0,0 +1,253 @@
+// -*- mode: rust; -*-
+//
+// This file is part of curve25519-dalek.
+// Copyright (c) 2016-2019 Isis Lovecruft, Henry de Valence
```

Comment:
Here and in the other new files you create I think it would be better to list yourself (the author of the new files) as the copyright holder.

Reply:
ACK; I'll fix that as I rework the PR.